README.md

```diff
@@ -192,9 +192,22 @@ curl -s localhost:7000/v1/appdata/ <block-number>
 Result:
 
 ```json
-{"block":386,"extrinsics":[{"app_id":1,"signature":{"Sr25519":"be86221cc07a461537570637d75a0569c2210286e85c693e3b31d94211b1ef1eaf451b13072066f745f70801ad6af0dcdf2e42b7bf77be2dc6709196b4d45889"},"data":"0x313537626233643536653339356537393237633664"}]}
+{"block":386,"extrinsics":["{hex_encoded_data}"]}
```

Comment:
IMO, the prev example was better as it showed that it included signature as well as the hex encoded data. Also to note, IIRC, the hex encoded data was not the submitted data but the extrinsic calldata.

Reply:
You are right, its a bad wording. But what to put as example since it returns hex encoded extrinsic and you have to decode it to get scheme as in this example?